### DISCUSSION TITLE
Placing objects on ceiling

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#2930] [General]
Hi everyone, 

Are we able to place objects on the ceiling without having the object flip in Model Viewer? 

https://user-images.githubusercontent.com/90825510/139180662-1fa56316-436a-452f-94bc-419a7e825c50.mp4

?

**Comment by elalish** [comment]
That's hilarious. That's Apple's Quick Look AR mode you're showing, so you'll need to file a bug on them. My best guess is to file it here: https://bugs.webkit.org/

I can't seem to repro on my iphone, but it might just be because it can't find my ceiling.

**Comment by aviggiano** [comment]
@Jasonp8000 how did you manage to achieve that effect? I can't repro either

**Reply by Jasonp8000** [reply]
Simply dragged the object on the ceiling and it automatically flipped the object. I'm using an iPhone 12 Pro Max - iOS 15 if that helps

**Comment by Jasonp8000** [comment]
Hi everyone, what's the result when placing an object on the ceiling for you guys ? @aviggiano  @elalish. Does the object snap to the ceiling and which devices are you using  ?

**Reply by aviggiano** [reply]
I can't do it. I'm using an iPhone X with iOS 15.0.2

**Reply by ghost** [reply]
I was able to repo it on my iPad Pro iOS 15.1
I took lots of panning back and forth on the ceiling to generate the surface but it worked.

**Reply by aviggiano** [reply]
Maybe this feature requires a LiDAR Scanner?

**Reply by ghost** [reply]
🤔  Actually this particular iPad doesn't have LiDAR

**Comment by buddy90210** [comment]
Can I place items on the ceiling in WebXR mode or scene viewer?

**Reply by elalish** [reply]
Not currently; feel free to open a feature request, or better yet, a PR.
